terraform/modules/enclave/prometheus/main.tf

```diff
@@ -4,6 +4,7 @@ terraform {
 
 locals {
   enable_public_ip = "${var.enable_ssh == 1 ? true : false}"
+  filebeat_count   = "${var.logstash_hosts != "" ? 1 : 0}"
```

Comment:
urgh terraform

Reply:
Indeed. Interesting conversation here about the direction Terraform want this to go https://github.com/hashicorp/terraform/issues/18195